### DISCUSSION TITLE
How do you use the call macro on an ActorCell?

### DISCUSSION THREAD

**Discussion by lushmana** [slawlor/ractor#122] [Q&A] [answered]
I want to be able to use call! Macro on an ActorCell, using the registry::where_is to get a registered actor, then work with the actor using a call macro. I can do through an ActorRef, but I need to use the registry to get an ActorRef.

**Comment by slawlor** [comment, marked as answer]
>  I can do through an ActorRef, but I need to use the registry to get an ActorRef.

You need to first cast the cell to a ref with .into() of the correct message type. The macros are all built around actorrefs, not cells presently.
